arch/src/lib.rs

```diff
@@ -117,11 +117,11 @@ pub struct NumaNode {
 
 pub type NumaNodes = BTreeMap<u32, NumaNode>;
 
-/// Type for passing information about the initramfs in the guest memory.
-pub struct InitramfsConfig {
-    /// Load address of initramfs in guest memory
+/// Type for passing information about the initramfs/kernel in the guest memory.
+pub struct BootFileConfig {
```

Comment:
The name `BootFileConfig` sounds too generic, but I am not good at naming.

Any better idea for the name?